lsp-ui-doc.el

```diff
@@ -274,13 +296,66 @@ We don't extract the string that `lps-line' is already displaying."
    ((gethash "language" contents) ;; MarkedString
     (lsp-ui-doc--extract-marked-string contents))))
 
+(defun lsp-ui-doc--webkit-run-xwidget ()
+  "Launch embedded WebKit instance."
+  (lsp-ui-doc--with-buffer
+   (let ((inhibit-read-only t))
+     (insert " ")
+     (goto-char 1)
+     (let ((id (make-xwidget
+                'webkit
+                nil
+                1
+                1
+                nil
+                (buffer-name))))
+       (xwidget-resize id 1 1)
```

Comment:
Is this call to `xwidget-resize` necessary ? I see you're already initializing the widget with the same size